lib/indexer/records.js

```diff
@@ -0,0 +1,152 @@
+/*!
+ * records.js - indexer records
+ * Copyright (c) 2018, the bcoin developers (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+/**
+ * @module lib/records
+ */
+
+const bio = require('bufio');
+const util = require('../utils/util');
+const consensus = require('../protocol/consensus');
+
+/**
+ * Block Meta
+ * @alias module:indexer.BlockMeta
+ */
+
+class BlockMeta {
+  /**
+   * Create block meta.
+   * @constructor
+   * @param {Hash} hash
+   * @param {Number} height
+   */
+
+  constructor(hash, height) {
+    this.hash = hash || consensus.NULL_HASH;
```

Comment:
I cannot find any references to `NULL_HASH` in the codebase, I think you want `ZERO_HASH`